tests/mentionui.js

```diff
@@ -799,6 +799,39 @@ describe( 'MentionUI', () => {
 			} );
 		} );
 
+		describe( 'unicode', () => {
+			beforeEach( () => {
+				return createClassicTestEditor( {
+					feeds: [
+						{
+							// Always return 5 items
+							feed: () => [ '@Barney', '@Lily', '@Marshall', '@Robin', '@Ted' ],
+							marker: '@'
+						}
+					]
+				} );
+			} );
+
+			it( 'should open panel for unicode character ב', function() {
+				if ( !featureDetection.isUnicodeGroupSupported ) {
+					this.skip();
+				}
+
+				setData( model, '<paragraph>foo []</paragraph>' );
+
+				model.change( writer => {
+					writer.insertText( '@ב', doc.selection.getFirstPosition() );
```

Comment:
How does it work? Assertions show that the UI shows up but `feed: () => [ '@Barney', '@Lily', '@Marshall', '@Robin', '@Ted' ],` contains no hebrew characters.

Reply:
Hmmm that's weird - I'll check it.

Reply:
Ugh.. the feed is a callback so it does not filter - it will always return the same list. I'll fix the test setup so it will not be so confusing.

Reply:
OK, fixed.